data-service/src/main/java/uk/gov/gchq/palisade/service/data/model/AuditableDataResponse.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2018-2021 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.palisade.service.data.model;
+
+import uk.gov.gchq.palisade.Generated;
+
+import javax.annotation.Nullable;
+
+import java.util.Objects;
+import java.util.Optional;
+import java.util.StringJoiner;
+
+/**
+ * The class contains the audit message from the processing of the request.  This will be a {@link AuditSuccessMessage}
+ * or possibly an {@link AuditErrorMessage} message generated in the processing of the request.
+ * */
+public final class AuditableDataResponse {
```

Comment:
Again, might be better to rename this class to something like this:
```suggestion
public final class AuditableReaderResponse {
```

Reply:
Gone the other approach using the keyword "authorised" to make the distinction.